tests/unit/ContactMethodsPageTest.tsx

```diff
@@ -0,0 +1,92 @@
+import {render, screen} from '@testing-library/react-native';
+import React from 'react';
+import Onyx from 'react-native-onyx';
+import type {ValueOf} from 'type-fest';
+import ComposeProviders from '@components/ComposeProviders';
+import ContactMethodsPage from '@pages/settings/Profile/Contacts/ContactMethodsPage';
+import DelegateNoAccessModalProvider from '@src/components/DelegateNoAccessModalProvider';
+import LockedAccountModalProvider from '@src/components/LockedAccountModalProvider';
+import type CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
+
+// Mock navigation used by the page
+jest.mock('@libs/Navigation/Navigation', () => ({
+    navigate: jest.fn(),
+    goBack: jest.fn(),
+    getActiveRoute: jest.fn(() => ''),
+}));
+
+// Replace MenuItem with a simple test double that exposes props in the tree
+jest.mock('@components/MenuItem', () => {
+    const ReactMock = require('react') as typeof React;
+    const {Text} = require('react-native') as {Text: React.ComponentType<{testID: string; children?: React.ReactNode}>};
+    return ({title, brickRoadIndicator}: {title: string; brickRoadIndicator?: ValueOf<typeof CONST.BRICK_ROAD_INDICATOR_STATUS>}) =>
+        ReactMock.createElement(Text, {testID: `menu-${String(title)}`}, `${brickRoadIndicator ?? 'none'}-brickRoadIndicator`);
+});
+
+describe('ContactMethodsPage', () => {
+    beforeAll(() => {
+        Onyx.init({
+            keys: ONYXKEYS,
+        });
+    });
+
+    beforeEach(() => {
+        return Onyx.clear();
+    });
+
+    function renderPage() {
+        return render(
+            <ComposeProviders components={[LockedAccountModalProvider, DelegateNoAccessModalProvider]}>
+                {/* @ts-expect-error - route typing is not necessary for this test */}
+                <ContactMethodsPage route={{params: {}}} />
+            </ComposeProviders>,
+        );
+    }
+
+    it('sets error indicator when login has error fields', async () => {
```

Comment:
I believe there should also be RBR/GBR on the bottom tabs, could you please test for that?